### ISSUE TITLE
7 day power graphs all blank

### ISSUE THREAD

**Issue by OutbackMatt** [Ylianst/MeshCentral#1322]
MeshCentral Version 0.5.27

All devices
7 day power graph is empty 

![Capture](https://user-images.githubusercontent.com/7231493/81458507-091ec480-91de-11ea-8902-fd53f1a4f979.GIF)

**Comment by Ylianst** [comment]
Not seeing this on my servers. Can you open the browser console and see if there are any errors? - Thanks.

**Comment by vitko-bg** [comment]
Just checked on my also - all looks fine. I am with same version 0.5.27 updated recently

**Comment by Ylianst** [comment]
Other things to check is if running NeDB and the "meshcentral-power.db" file is correct in "meshcentral-data", and there is space on the drive. If using MongoDB, look at the "power" collection for anything wrong.

**Comment by Ylianst** [comment]
No feedback on this one in a while. I will close it. Please re-open or file a new issue if needed. Thanks.

**Comment by OutbackMatt** [comment]
Just a follow up on this
(I got busy with Corona stuff sorry - I work in Healthcare)
At the time I did completely remove the power collection, even though I could see nothing wrong and hoped that it would start working again.

Nothing happened, all graphs remained blank, until 

suddenly they started to work about new year.


Don't recall exactly when, but all graphs are back to normal now, showing 7 day power state.

<shrugs shoulders>
